akka-stream/src/main/scala/akka/stream/javadsl/Sink.scala

```diff
@@ -136,6 +136,19 @@ object Sink {
       _.map(akka.japi.Option.fromScalaOption)(ExecutionContexts.sameThreadExecutionContext)))
 
   /**
+   * A `Sink` that keeps on collecting incoming elements until upstream terminates.
+   * As upstream may be unbounded, `Flow[T].take` or the stricter ``Flow[T].limit` (and their variants)
+   * may be used to ensure boundedness.
+   * Materializes into a Future` of `Seq[T]` containing all the collected elements.
```

Comment:
the backticks look unbalanced in this comment -- does it render OK in the Scaladoc?